### ISSUE TITLE
Moto Z3 Play Battery Drain

### ISSUE THREAD

**Issue by nmajin** [PixelExperience/android-issues#2749]
## Build date

PixelExperience_beckham-12.1-20220523-1042-OFFICIAL.zip

## Expected Behavior

Phone should not drain so fast when idle.

## Current Behavior

Phone is draining faster than normal. System usage is showing "Mobile network" consuming the most battery by far. This is a clean install too, done a couple times with the same behavior.

## Possible Solution

I don't know, you tell me. Maybe someone smarter than me can discover the bug  :)

## Steps to Reproduce

1. Flash PE 12 recovery
2. Format and then sideload a/b copy
3. Format and thehn sideload retrofit
4. Boot retrofit once, go back to recovery and format then install PE  12 build.

/device beckham
/version twelve

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by nmajin** [comment]
To add, I am only using Wifi, no SIM card installed.

**Comment by nmajin** [comment]
I just updated to the June release so I will test this out on a full charge.

**Comment by jro1979oliver** [comment]
Any input on this issue? Some users reported that turning roaming off makes telephony less agressive on battery drain

**Comment by nmajin** [comment]
Same issue. I will try turning off roaming and see if that makes a difference.

**Comment by nmajin** [comment]
Forgot to mention I don't have a SIM in the phone so Roaming is already not on. It seems like "Mobile network" is still the high consumer of my battery, drops about 5-8% per hour with the phone just idle and not being used. There are no additional apps installed and it is a complete fresh install with the July update.

**Comment by tim1aust** [comment]
Same issue with latest PE 13 Plus on beckham. CPU never goes below 48%. I also have no SIM, just using WiFi. I have turned off 'Mobile data always active' in developer settings, and high battery drain from Mobile network has dropped dramatically, but I still have high CPU (48-80%) at idle and phone stays warm.  I have a Moto Mod battery pack attached to try and keep the phone battery life to a full day. Other than this issue, this ROM is great and I am very appreciative that folks in this community have allowed me to extend the life of my much-loved Z3 Play :)

**Comment by tim1aust** [comment]
Update to my post from yesterday. I am using Resource Monitor Mini Pro to display CPU usage. I tried 3C CPU manager yesterday to try and dig in deeper as to which process was using the CPU, and it showed much lower CPU usage. So today I uninstalled and reinstalled my Resource Monitor Mini and now it is showing much more reasonable CPU usage at idle (20 - 30% or so).  So I apologize for jumping to conclusions about the ROM -- it appears to have been the app I was using to monitor CPU usage.  Thanks again for this great ROM on the Z3 Play.
